pom.xml

```diff
@@ -25,7 +25,8 @@
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongo-java-driver</artifactId>
-            <version>2.13.0</version>
+            <version>3.0.0</version>
+            <scope>provided</scope>
```

Comment:
Yes: provided so that projects using either 2.x or 3.0 can depend on one release of mongojack.